can/broadcastmanager.py

```diff
@@ -250,6 +252,8 @@ def _run(self):
                     self.bus.send(self.messages[msg_index])
                 except Exception as exc:
                     log.exception(exc)
+                    if self.error_callback:
+                        self.error_callback(exc)
```

Comment:
Maybe you want to conditionally break on `error_callback` return?

This way, the error handler (`error_callback`) can decide if this is a catastrophic error and you need to stop sending or if this can be recovered automatically.

Reply:
I am not sure. I thought about that. In such case, resources are wasted by the thread responsible to send periodic messages while temporary error persists, personally, I prefer to handle that situation by other means outside of this loop, and restart periodic sender once issue resolves.

However if you want it to work that way, I can make such changes.

Reply:
@pierreluctg I did proposed changes. Can you please have a look?

Reply:
looks good to me